src/engraving/dom/range.cpp

```diff
@@ -844,3 +899,68 @@ void TrackList::dump() const
     }
 }
 }
+
```

Comment:
This brace id wrong here, needs to be at the any, so the following code is in the same namespace